### DISCUSSION TITLE
Why Can't I Control Devices From My RustDesk Server?

### DISCUSSION THREAD

**Discussion by MerryPickles** [rustdesk/rustdesk-server-pro#155] [General]
If I run RustDesk on any computer in my office's network, the same network on which my  RustDesk server is installed, then I cannot use it to control any of the devices I set up. I can control them from any other computer that is outside of my office's network. If I use my phone as a hotpsot, then no problem. Any ideas?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#deploy-rustdesk-server-in-intranet
